src/ua/burkavtsov/hw11/ArrayList.java

```diff
@@ -0,0 +1,70 @@
+package ua.burkavtsov.hw11;
+
+import java.util.Objects;
+
+public class ArrayList<T> {
+    private Object[] array;
+    private int size;
+
+    public ArrayList() {
+        this.size = 0;
+        this.array = new Object[10]; /
+    }
+
+    public void add(T element) {
```

Comment:
If we have beginning size of array is bigger than 0 this method will work incorrect